Netskope/netskope_events/ingest/parser.yml

```
@@ -144,15 +144,51 @@ stages:
           netskope.events.application.activity: "{{parsed_event.message.app_activity}}"
           netskope.events.access_method: "{{parsed_event.message.access_method}}"
           netskope.events.ccl: "{{parsed_event.message.ccl}}"
-          netskope.events.severity.id: "{{parsed_event.message.severity_id}}"
-          netskope.events.severity.level: "{{parsed_event.message.severity_level}}"
+
+      - set:
+          netskope.events.severity_id: "{{parsed_event.message.severity_id}}"
+          netskope.events.severity_level: "{{parsed_event.message.severity_level}}"
+        filter: "{{ parsed_event.message.severity_level|int(-1) == -1 }}"
+
+      - set:
+          netskope.alerts.severity: "{{parsed_event.message.severity}}"
+        filter: "{{ parsed_event.message.severity|int(-1) == -1}}"
+
+      - set:
+          netskope.audit.severity_id: "{{parsed_event.message.severity_level}}"
+        filter: '{{ parsed_event.message.severity_level|int(-1) >= 0 and parsed_event.message.type == "admin_audit_logs"}}'
+
+      - translate:
+        dictionary:
+          1: "High"
+          2: "Medium"
+          3: "Low"
+        mapping:
+          parsed_event.message.severity_level: netskope.audit.severity_level
+        fallback: "Info"
+        filter: '{{ parsed_event.message.severity_level|int(-1) >= 0 and parsed_event.message.type == "admin_audit_logs"}}'
+
```

My thought was to separate severity_level for "admin_audit_logs" and "malsite" events by using a different custom field for the latter - because it represenets severity of the site, not the event itself. While for "admin_audit_logs" severity_level is describing event itself. And it's very confusing.

Also, it seems that for malsite events we get 'med' instead of 'medium'.
So, I'd go with something like this:

```suggestion
      - translate:
         dictionary:
           "med": "Medium"
           "high: "High"
           "low": "Low"
         fallback: "Info"
         mapping:
           parsed_event.message.severity_level: netskope.audit.malsite_severity_level
         filter: "{{ parsed_event.message.severity|int(-1) == -1 and parsed_event.message.type == "malsite" }}"
        
      - translate:
        dictionary:
          1: "High"
          2: "Medium"
          3: "Low"
        mapping:
          parsed_event.message.severity_level: netskope.audit.severity_level
        fallback: "Info"
        filter: '{{ parsed_event.message.severity_level|int(-1) >= 0 and parsed_event.message.type == "admin_audit_logs"}}'                        
```